src/Tests/Equinor.Procosys.Library.Query.Tests/GetAllAreas/GetAllAreasQueryTests.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Equinor.Procosys.Library.Query.GetAllAreas;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Equinor.Procosys.Library.Query.Tests.GetAllAreas
```

Comment:
add test asserting for ArgumentNullException when plant is null

Reply:
Do we actually need to fail if a plant name is wrong/empty? Wouldn't it be ok to just return an empty response?